mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMailboxManager.java

```diff
@@ -461,7 +461,7 @@ public MessageManager getMailbox(MailboxId mailboxId, MailboxSession session)
             throw new MailboxNotFoundException(mailboxId);
         }
 
-        if (! assertUserHasAccessTo(mailboxRow, session)) {
```

Comment:
I removed that diff from my PR as it broke things.... (case on users...)

Could you do the same?

https://github.com/linagora/james-project/pull/1085/commits/2bf397dd86617202c78c5ae7d57fb9c93cdbfd02

 MAILBOX-317 Bonus: Simplify a bit right handling in StoreMailboxManager

Reply:
Remove that commit.